src/HypercertExchangeClient.ts

```diff
@@ -209,7 +201,7 @@ export class HypercertExchangeClient {
       endTime: endTime,
       price: price,
       itemIds: itemIds,
-      amounts: amounts,
+      amounts: itemIds.map(_ => 1n),
```

Comment:
Does this make sense? is it always 1? Will this break when we enable other strategies?